packages/react-components/react-swatch-picker-preview/src/components/ColorSwatch/renderColorSwatch.tsx

```diff
@@ -10,5 +10,11 @@ import type { ColorSwatchState, ColorSwatchSlots } from './ColorSwatch.types';
 export const renderColorSwatch_unstable = (state: ColorSwatchState) => {
   assertSlots<ColorSwatchSlots>(state);
 
-  return <state.root />;
+  return (
+    <state.root>
+      {state.root.children}
+      {state.icon && <state.icon />}
+      {state.disabled && state.disabledIcon && <state.disabledIcon />}
```

Comment:
Having together icons vs initials and disabled state will be discussed today with the designer. If we decide to change something I'll update it in the following PRs